app/api/schema/custom_forms.py

```diff
@@ -3,8 +3,10 @@
 from marshmallow_jsonapi.flask import Schema, Relationship
 
 from app.api.helpers.utilities import dasherize
+from utils.common import use_defaults
 
 
+@use_defaults()
```

Comment:
Looks good. Remove paranthesis, as they are not needed

Reply:
It need ```self``` right ?